### PR TITLE
Update local monitoring demos with multiple metadata imports

### PR DESCRIPTION
**Pull request by shreyabiradar07**
## Description

This PR updates local monitoring demo script showcasing multiple metadata import actions adding below steps:

- Delete previously imported metadata
- List Metadata to ensure metadata database records are deleted successfully
- Import metadata from prometheus-1 datasource 
- Display metadata from prometheus-1 datasource 
- Create new namespace and deploy an app in new namespace created
- Multiple import metadata action in the same prometheus-1 datasource
- Display metadata from prometheus-1 datasource 
- Create kruize experiments - for the new containers with different experiment names
- Generate recommendations for the new experiments created

Local monitoring demo execution on `minikube`: https://privatebin.corp.redhat.com/?efc4bba2780cb09d#5NrFVVVx3ksogT2WJaXjP1QtMMH1S2mZPQbrZu4b5EeL

on `Openshift`: https://privatebin.corp.redhat.com/?11f2190fd5439274#CC2qpvigvJPAF2F3HEnUpH6frAKDmBmrwTED9dRfhmPc

### Type of change

- [ ] Bug fix
- [ ] New feature
- [ ] Docs update
- [x] Enhancement

## How has this been tested?
**Test Configuration**
* Kubernetes clusters tested on: minikube and openshift

## Checklist :dart:

- [x] Followed coding guidelines
- [x] Comments added
- [ ] Dependent changes merged
- [ ] Documentation updated
- [x] Tests added or updated

## Additional information

Docker image : ` quay.io/shbirada/multiple_import:kl`

### CODE REVIEW

File: monitoring/local_monitoring/local_monitoring_demo.sh [hunk 1]
```diff
@@ -141,6 +141,88 @@ function kruize_local() {
 	echo
 	echo "######################################################"
 	echo
+
+	echo
+  echo "######################################################"
+  echo "#     Delete previously imported metadata"
+  echo "######################################################"
+  echo
+  curl -X DELETE http://"${KRUIZE_URL}"/dsmetadata \
```
Comment (chandrams) [inline monitoring/local_monitoring/local_monitoring_demo.sh:150]: @shreyabiradar07 - You can remove delete metadata. Deploy another app into a new namespace and do an import metadata

Comment (shreyabiradar07) [reply monitoring/local_monitoring/local_monitoring_demo.sh:150]: Have retained delete `/dsmetadata` API call to showcase `DELETE` functionality and added additional import metadata `POST /dsmetadata`  calls to showcase multiple import metadata functionality by creating a new namespace and deploying TFB benchmark in the new namespace created

Updated demo script execution output - https://privatebin.corp.redhat.com/?efc4bba2780cb09d#5NrFVVVx3ksogT2WJaXjP1QtMMH1S2mZPQbrZu4b5EeL

Comment (chandrams) [reply monitoring/local_monitoring/local_monitoring_demo.sh]: @shreyabiradar07 
I still see the older error message in the output, wasn't this fixed to just have the parameters that were passed to list_dsmetadata?
{"message":"Metadata for a given datasource - \" prometheus-1 \", cluster name - \" null \", namespace - \"null \" either does not exist or is not valid","httpcode":400,"documentationLink":"","status":"ERROR"}

Also, I don't see any cpu recommendations in the output. Can you check this.

Comment (shreyabiradar07) [reply monitoring/local_monitoring/local_monitoring_demo.sh]: > I still see the older error message in the output, wasn't this fixed to just have the parameters that were passed to list_dsmetadata?
> {"message":"Metadata for a given datasource - " prometheus-1 ", cluster name - " null ", namespace - "null " either does not exist or is not valid","httpcode":400,"documentationLink":"","status":"ERROR"}

Error message is fixed as part of #[1200](https://github.com/kruize/autotune/pull/1200) which is yet to be merged 

> Also, I don't see any cpu recommendations in the output. Can you check this.

The above output shared is by running the demo script on `minikube`. Will run the script on `openshift` and share the updated output

Comment (shreyabiradar07) [reply monitoring/local_monitoring/local_monitoring_demo.sh]: @chandrams Demo script output on `openshift` - https://privatebin.corp.redhat.com/?11f2190fd5439274#CC2qpvigvJPAF2F3HEnUpH6frAKDmBmrwTED9dRfhmPc

Comment (chandrams) [reply monitoring/local_monitoring/local_monitoring_demo.sh]: @shreyabiradar07 - I think with minikube certain versions have issues, you can try with kind. Openshift output looks fine.

**Review by chandrams — APPROVED** [review]
LGTM

File: common/common_helper.sh [hunk 2]
```diff
@@ -294,3 +294,23 @@ function check_minikube() {
...
+###########################################
+#   Deploy TFB Benchmarks - multiple import
+###########################################
+function create_namespace_and_install_benchmarks() {
+	echo
+	echo "#######################################"
+	echo "Creating new namespace: test-multiple-import"
+  	kubectl create namespace test-multiple-import
+	echo "#######################################"
+	pushd benchmarks >/dev/null
```
Comment (kusumachalasani) [inline common/common_helper.sh:306]: @shreyabiradar07 section of code which installs benchmarks looks duplicated. Would be good if you can use the existing benchmarks_install function and only have create_namespace as a different function.
You can modify the existing benchmarks_install function to use the namespace.

Comment (shreyabiradar07) [reply common/common_helper.sh]: @kusumachalasani removed the duplicate code to install benchmarks, adding custom namespace support and also updated the manifests to `manifests/default_manifests`

**Pull request merged**